buckw

```diff
@@ -213,10 +213,15 @@ setupBuckRun ( ) {
 
 # Handle parameters and flags
 handleParams ( ) {
-    # Go directly to kill. Do not run okbuck.
-    if [[ "kill" == $1 ]]; then
-        SKIP_OKBUCK=true
-    fi
+    for arg in "$@"
```

Comment:
We can also use `comm` to see if there is an intersection of arguments we want to check for. See https://stackoverflow.com/a/2696111 for example. That would make it very easy to add new arguments